pkgs/applications/graphics/megapixels/default.nix

```diff
@@ -0,0 +1,45 @@
+{ stdenv
+, lib
```

Comment:
```suggestion
{ stdenv
```

Please only use one.

Reply:
Why, is that a new packaging role? There's plenty of existing packages in Nixpkgs that use both for convenience.

```
↪ grep -rin "{ stdenv, lib," pkgs/ | wc -l
631
```

Reply:
Gonna revert that change, see #108938, #108978